practice/src/css/sections/package.css

```diff
@@ -0,0 +1,111 @@
+.wrapper-package {
+  padding-bottom: 50px;
+}
+  
+.package-card-item {
+  position: relative;
+
+  &:hover {
+    border-radius: var(--primary-border-radius);
+    border: 4px solid var(--primary-border-radius-color);
+    transition: transform 0.3s ease;
+    transform: scale(0.98);
+    padding: 5px;
+  }
+}
+
+.package-card-item-image {
+  border-radius: var(--primary-border-radius);
+  width: 100%;
+  height: 380px;
+}
+  
+.package-information {
+  position: absolute;
+  bottom: 35px;
+  left: 40px;
+}
+  
+.package-icon {
+  padding-bottom: 5px;
+}
+  
+.package-address {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+}
+  
+.package-introduction {
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+  
+.package-evaluation {
+  display: flex;
+  gap: 10px;
+  padding-top: 30px;
+  padding-bottom: 25px;
+}
+  
+.introduction-desc {
+  padding-top: 10px;
+}
+  
+.package-desc {
+  color: var(--gray-text-color);
+}
+
+.package-title {
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-medium);
+  line-height: var(--secondary-line-height);
+  color: var(--primary-text-color);
+  display: flex;
```

Comment:
@JennyDoJD I am not sure why package-title should be `display: flex` ? could you help explain?

Reply:
I added it in the wrong place